modules/nux/setup.php

```diff
@@ -75,6 +79,9 @@
         'nux_name' => FILTER_DEFAULT,
         'nux_pass' => FILTER_UNSAFE_RAW,
         'nux_account_name' => FILTER_DEFAULT,
-        'nux_all_inkl_login' => FILTER_DEFAULT
+        'nux_all_inkl_login' => FILTER_DEFAULT,
+        'accounts_source' => FILTER_DEFAULT,
+        'accounts_source' => FILTER_DEFAULT,
+        'accounts_csv' => FILTER_DEFAULT,
     )
```

Comment:
so it is csv, yaml or both?

Reply:
This on is using yaml for now. Should i add csv too or just remove any mention of it ??

Reply:
@TheDigitalOrchard You suggested YAML. What do you think?

Reply:
Any thought about both yaml and csv ?

Reply:
> both yaml and csv ?

I like it!

Reply:
CSV and yaml are now both supported.